servers/push-server/src/worker.ts

```diff
@@ -23,40 +19,13 @@ const processMessage = async (fullMessage: Message): Promise<void> => {
     level: 'info',
   });
 
-  if (target === TARGET_APNS) {
-    console.log('APNS push', token);
-    await apns.sendNotificationToDevice(
-      contents,
-      {
-        s: message.share_id,
-        i: message.item_id,
-        url: message.url,
-        notification_id: message.notification_id,
-      },
-      token,
-      false,
-    );
-  } else if (target === TARGET_APNS_SILENT) {
+  if (target === TARGET_APNS_SILENT) {
     console.log('APNS SILENT push', token);
-    await apns.sendNotificationToDevice(
-      contents,
-      {
-        g: message.guid,
```

Comment:
Clients don't rely on the guid payload anymore. (verified in android and ios codebase)

Reply:
It also simplifies the required code here.